Sources/CorePayments/Eligibility/EligibilityIntent.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
```

Comment:
I added this enum from the logic that was previously removed: https://github.com/paypal/paypal-ios/blob/257df315bb8ab2e323df39bf17a5af48e36498a1/Sources/CorePayments/Networking/GraphQL/FundingEligibilityIntent.swift#L1